queries.sql

```diff
@@ -25,4 +25,32 @@ SELECT * FROM animals WHERE name != 'Gabumon';
 
 
 //animals with a weight between 10.4kg and 17.3kg
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+//How many animals are there?
+SELECT COUNT(name) FROM animals;
+
+//How many animals have never tried to escape?
+SELECT COUNT(escape_attempts) FROM animals
+WHERE escape_attempts = 0;
+
+//What is the average weight of animals?
+SELECT AVG(weight_kg) FROM animals;
+
+
+//Who escapes the most, neutered or not neutered animals?
+SELECT MAX(escape_attempts) as max_escapes FROM animals WHERE neutered = 'true';
+SELECT MAX(escape_attempts) as max_escapes FROM animals WHERE neutered = 'false';
+
+
+
+What is the minimum and maximum weight of each type of animal?
+SELECT MAX(weight_kg) as max_Weigth FROM animals WHERE species = 'pokemon';
+SELECT MIN(weight_kg) as min_Weigth FROM animals WHERE species = 'pokemon';
+
+SELECT MAX(weight_kg) as max_Weigth FROM animals WHERe species = 'digimon';
+SELECT MIN(weight_kg) as min_Weigth FROM animals WHERe species = 'digimon';
```

Comment:
- Your query should select the species, min and max weight then groped by the species. right now, you've made 4 queries instead of 1 query. kindly re-evaluate the queries.